WCFDataService/Client/System/Data/Services/Client/Materialization/MaterializerEntry.cs

```diff
@@ -275,7 +275,7 @@ public static MaterializerEntry CreateEntryForLoadProperty(EntityDescriptor desc
         /// <returns>The materializer entry</returns>
         public static MaterializerEntry GetEntry(ODataEntry entry)
         {
-            return entry.GetAnnotation<MaterializerEntry>();
+            return entry == null ? null : entry.GetAnnotation<MaterializerEntry>();
```

Comment:
Hmmm. I could do that, but... If entry really is null and AutoNullPropagation is false - GetEntry itself would fail with an object null exception - I don't think the code would ever get in this situation as we would have seen it long time ago. Do you still think I should change this?